.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -1,38 +1,27 @@
 name: Wheel::Linux::ARM
+
+# **What it does**: Builds python wheels for Linux (ubuntu-latest) architecture ARM 64 and store it as artifacts.
+#                   Python versions: 3.7, 3.8, 3.9, 3.10.
+# **Why we have it**: To build wheels for pennylane-lightning instalation.
```

Comment:
Typo also appears throughout document I believe.
```suggestion
# **Why we have it**: To build wheels for pennylane-lightning installation.
```

Reply:
Thank you for mentioning it. I will look into this.